config/config.toml

```diff
@@ -1,5 +1,6 @@
 
 #reporting_disabled = false
+node_id = 100 
 
```

Comment:
Config in default file `config.toml` should be commented instead of omitted.

#### Example
```toml
# host = 'localhost'
```

Reply:
ok